internal/uvm/create_wcow.go

```diff
@@ -222,6 +223,23 @@ func prepareConfigDoc(ctx context.Context, uvm *UtilityVM, opts *OptionsWCOW, uv
 		}
 	}
 
+	// Set boot options
+	if opts.DumpDirectoryPath != "" {
```

Comment:
is this a windows-only feature?

Reply:
Not really. If the guest is a Linux VM that has hyperv drivers it should also generate a similar crash dump. But I think we already get enough information from dmesg logs to debug a Linux UVM crash. I was not able to crash a linux VM in such a way that it would generate a crash. So I will keep this only for WCOW for now. Is that okay?

Reply:
sure thing.